.travis.yml

```diff
@@ -1,14 +1,26 @@
 language: cpp
 matrix:
   include:
-    - name: "ARCH=arm"
-      env: ARCH=arm
-    - name: "ARCH=arm64"
-      env: ARCH=arm64
-    - name: "ARCH=arm64 LD=ld.lld"
-      env: ARCH=arm64 LD=ld.lld-8
-    - name: "ARCH=x86_64"
-      env: ARCH=x86_64
+    - name: "ARCH=arm REPO=linux"
+      env: ARCH=arm REPO=linux
+    - name: "ARCH=arm64 REPO=linux"
+      env: ARCH=arm64 REPO=linux
```

Comment:
I feel like with these building blocks (conditional on cron) we could make non-lld arm64 builds cron only.

Idea being that on our free tier, we only have access to three concurrent builds. So to to have fast presubmit feedback, we want to minimize our PR builds to multiples of three (preferably only three).

Is that a bad idea? 🎲 📍

Reply:
I wouldn't be opposed to this. I can either do that in this pull or another one, up to you.